.scala-steward.conf

```diff
@@ -1,11 +1,13 @@
 updates.ignore = [
   // explicit updates
   { groupId = "com.typesafe.akka", artifactId = "akka-actor" },
-  { groupId = "com.typesafe.akka", artifactId = "akka-persistence:2.5.25" },
-  { groupId = "com.typesafe.akka", artifactId = "akka-persistence-query:2.5.25" },
-  { groupId = "com.typesafe.akka", artifactId = "akka-stream:2.5.25" },
+  { groupId = "com.typesafe.akka", artifactId = "akka-persistence" },
+  { groupId = "com.typesafe.akka", artifactId = "akka-persistence-query" },
+  { groupId = "com.typesafe.akka", artifactId = "akka-stream" },
+  { groupId = "com.typesafe.akka", artifactId = "akka-slf4j" },
+  { groupId = "com.typesafe.akka", artifactId = "akka-persistence-tck" },
+  { groupId = "com.typesafe.akka", artifactId = "akka-stream-testkit" },
+  { groupId = "com.typesafe.akka", artifactId = "akka-testkit" },
   // inherited from Akka
   { groupId = "com.typesafe", artifactId = "config" }
 ]
-
-updatePullRequests = false
```

Comment:
@ennru, I merged this PR, but didn't see that `updatePullRequests` was added and then removed. This is a usefull thing and I believe was the initial Marcos' intention. 

I think we should add it back.